exporter/exporterhelper/internal/batch_sender.go

```diff
@@ -147,19 +142,26 @@ func (bs *BatchSender) Send(ctx context.Context, req internal.Request) error {
 	}
 
 	if bs.cfg.MaxSizeItems > 0 {
-		return bs.sendMergeSplitBatch(ctx, req)
+		return bs.sendMergeSplitBatch(ctx, req.(internal.BatchRequest))
 	}
-	return bs.sendMergeBatch(ctx, req)
+	return bs.sendMergeBatch(ctx, req.(internal.BatchRequest))
```

Comment:
Custom Request not implementing batching function would panic here. Let's return an error instead?

Reply:
It is ok for the moment to panic I believe since we are experimental. We will make it required to implement this to setup batching soon.

Reply:
@sfc-gh-sili for the moment move the functions on Request.

Reply:
@bogdandrutu Thanks!
What did you mean by move the functions on Request? Do you mean instead of having Request and BatchRequest, we would like to have Merge() and MergeSplitBatch() as a required function and then just have Request?

Reply:
>For users who do not do batching they can leave Merge() and MergeSplit() as noop I assume?

Correct.